src/pages/quizzes.tsx

```diff
@@ -0,0 +1,281 @@
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="basics" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="basics-description" />
+                </Text>
+
+                <QuizzesList
+                  content={ethereumBasicsQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Box px={{ base: 8, lg: 0 }} mb={10}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="using-ethereum" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="using-ethereum-description" />
+                </Text>
+
+                <QuizzesList
+                  content={usingEthereumQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Flex
+                direction={{ base: "column", lg: "row" }}
+                justifyContent="space-between"
+                alignItems="center"
+                bg="ednBackground"
+                borderRadius={{ base: "none", lg: "lg" }}
+                border="none"
+                p={{ base: 8, lg: 12 }}
+              >
+                <Stack mb={{ base: 4, lg: 0 }}>
+                  <Text
+                    align={{ base: "center", lg: "left" }}
+                    fontWeight="bold"
+                    color="body"
+                    mb={-2}
+                  >
+                    <Translation id="want-more-quizzes" />
+                  </Text>
+
+                  <Text align={{ base: "center", lg: "left" }} color="body">
+                    <Translation id="contribute" />
+                  </Text>
+                </Stack>
+
+                <ButtonLink
+                  to={"/contributing/learning-quizzes/"}
+                  variant="outline"
+                  hideArrow
+                  mt={0}
+                >
+                  <Flex alignItems="center">
+                    <Icon as={FaGithub} color="text" boxSize={6} me={2} />
+                    <Translation id="add-quiz" />
+                  </Flex>
+                </ButtonLink>
+              </Flex>
+            </Box>
+
+            {/* quizzes stats */}
+            <QuizzesStats />
+          </Flex>
+        </Box>
+      </QuizzesHubContext.Provider>
+
+      <Content>
+        <FeedbackCard />
+      </Content>
+    </Box>
+  )
+}
+
+export default QuizzesHubPage
+
+export const query = graphql`
+  query QuizzesHubPage($languagesToFetch: [String!]!) {
```

Comment:
Translation banner should not show in English

<img width="1574" alt="Screenshot 2023-05-29 at 09 42 31" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/a1e986ac-1588-408a-81a8-b7ceb3f20718">

Reply:
I think this a current bug we have in production, not related to this page cc @pettinarip

Reply:
#10368